addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1965,6 +1965,34 @@ options.registry.topMenuColor = options.Class.extend({
     },
 });
 
+options.registry.MobileVisibility = options.Class.extend({
+    isTopOption: true,
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+    * @override
+    */
+    async start() {
+        await this._super(...arguments);
+        this.$el.find('we-button').attr('title', this.$target.hasClass('d-md-block d-none') ? _t("Hidden on mobile") : _t("Visible on mobile"));
```

Comment:
Ok, reading the code I really was wondering how this line could work so I did test (like you should have) and it indeed does not work... hide a section, save the editor, go back to edit, check the title on the section... it says "visible" :/

`hasClass` from jQuery expects *one* class to be given. If you give mutliples ones, they have to be in the right order (bad luck you gave them in the inverse order that the one we use). But you cannot rely on order anyways, use two hasClass.

Anyways, there are two more problems with this:
1. This is duplicated code with what you do in `selectClass`
2. You duplicate the data, we have d-none d-md-block in the XML definition AND in the JS definition

There already is code which marks the icon as active or not (which you made green or not). The title should only depend on that and nothing else. Not duplicating the logic of "should it be active or not". What you should do is override the `_updateUI` method (which is what you want: updating the UI), which is naturally called on start and when changing an option. In `_updateUI` you just have to check if the button is active or not (no need to check classes on the target) and update the title accordingly.